src/css/reviewPage.css

```diff
@@ -0,0 +1,18 @@
+/* Review Page */
+.review-page__page-heading__text {
+  color: var(--color-primary-darker);
+}
+
+.review-page__page-heading__link {
+  text-align: right;
+  margin-top: 1.8em;
```

Comment:
If you could get close to this value by using one of the utils [here](https://design.va.gov/foundation/utilities/margins), you might be able to remove this entire file!

Reply:
@obliviga rad, thanks for all the helpful comments!